transports/msmq/index.md

```diff
@@ -78,13 +78,26 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 | Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
 | Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
 
-NOTE: If you are running in *Workgroup* mode then Windows will grant Send permissions to Anonymous. Everyone is added by default by Windows but not with Send or Receive permisssions. Prior to NServiceBus 6.1.0 Send permissions for Anonymous and Everyone were granted at queue creation but this behavior was altered to not accidentally open up permissions.
+
+For endpoints using the publish-subscribe pattern, both the publisher and the subscriber must have permission to send messages to each other. This is to allow the Subscribe message to be sent from the subscribing endpoint to the publishing endpoint.
+
+When an NServiceBus endpoint creates a queue on a machine, the default permissions depend on whether the machine is connected to a domain or a workgroup.
+
+### Domain mode
+
+If the machine is connected to a domain, then only the domain user running the endpoint when the queue is created will have Send permissions granted. If multiple endpoints need to communicate and are running under the same domain account, no further configuration is required. If multiple endpoints need to communicate using different domain accounts then the Send permission on the receiving endpoint input queue needs to be granted to the domain account of the sending endpoint.
+
+### Workgroup mode
+
+If the machine is connected to a workgroup then the Send permission is granted to the Everyone and Anonymous user groups by Windows. Any endpoint will be able to send messages to any other endpoint without further configuration.
+
+NOTE: In versions 5.2.20 and 6.0.3 a bug was fixes where [installers always changed permissions on a queue, also if it already existed](https://github.com/Particular/NServiceBus/issues/4266).
```

Comment:
This needs to be a new section/heading. Otherwise it looks to be a part of the Workgroup mode section.